templates/projects/notes/files/common/models/note.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "note",
```

Comment:
Ditto, it will be better to use `Note` as the name. (I do realise the original template used `note` instead.)